sample-app/src/icons/history.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="14" height="15" viewBox="0 0 14 15" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
In the component styling items, I removed the width and height from the SVGs I used and added a container around the SVG so that the sizing could be controlled with CSS. We may want to do that here as well to be consistent

Reply:
i see, updated!